.travis.yml

```diff
@@ -15,4 +15,6 @@ script:
   - cargo build --manifest-path boards/adafruit_nrf52pro/Cargo.toml --examples
   - cargo build --manifest-path boards/nRF52840-DK/Cargo.toml --examples
   - cargo build --manifest-path examples/rtfm-demo/Cargo.toml
-  - cargo build --manifest-path examples/rtfm-demo/Cargo.toml --no-default-features --features="52840"
\ No newline at end of file
+  - cargo build --manifest-path examples/rtfm-demo/Cargo.toml --no-default-features --features="52840"
+  - cargo build --manifest-path examples/spi-demo/Cargo.toml
```

Comment:
As is now the code is not prepared for 840 and 810.

Reply:
Oh, it somehow removed my comment when I clicked resolve.
Sorry, my brain somehow ignored that fact. Ofc in that case there is no need for those builds. I think the 810 has the same DK as the 832. No guarantees tho ;)

Reply:
It would be nice if it could run on both platforms, @Yatekii don't you have a 840 unit to try it on - If it only require trivial changes we could support both.

Reply:
Yes, I can test it at home. Not sure if I find time today, but seems doable!